kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/security/PlainSaslServer.java

```diff
@@ -43,6 +45,7 @@
 
     private boolean complete;
     private String authorizationId;
+    private String username;
```

Comment:
May need change this field to tenantSpec too?

Reply:
here it is the actual "username"

Reply:
is is about SASL auth here, we are not in the Pulsar/KOP domain yet

Reply:
You right, thanks for explain.